internal/config/config.go

```diff
@@ -89,9 +106,11 @@ func (c *Config) lookupByzStrategy(replicaID hotstuff.ID) string {
 	return ""
 }
 
+type ClienIdMap map[string][]hotstuff.ID
```

Comment:
Should we name it `ClientMap`? Should it too work with `orchestrationpb.ClientOpts` similar to `AssignReplicas`? Not sure...